src/main/scala/it/pagopa/interop/attributeregistryprocess/api/impl/AttributeRegistryApiServiceImpl.scala

```diff
@@ -125,11 +126,11 @@ final case class AttributeRegistryApiServiceImpl(
         )
         attributeSeedsCategories   = categories.map(c =>
           AttributeSeed(
-            code = Option(c.code),
+            code = Option(Digester.toSha256(c.kind.getBytes)),
             kind = AttributeKind.CERTIFIED,
-            description = c.name, // passing the name since no description exists at party-registry-proxy
+            description = c.kind, // passing the name since no description exists at party-registry-proxy
             origin = Option(c.origin),
-            name = c.name
+            name = c.kind
           )
```

Comment:
the attribute related to the kind should be an additional attribute, not a replacement.
The attribute should still be created as
```
AttributeSeed(
            code = Option(c.code),
            kind = AttributeKind.CERTIFIED,
            description = c.name, // passing the name since no description exists at party-registry-proxy
            origin = Option(c.origin),
            name = c.name
          )
```
in addition you need to create the attribute
```
AttributeSeed(
            code = Option(Digester.toSha256(c.kind.getBytes)),
            kind = AttributeKind.CERTIFIED,
            description = c.kind,
            origin = Option(c.origin),
            name = c.kind
          )
```

Note: it's better to perform a distinct on `categories.kind` before creating the attributes, because kinds are less than 10